cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Babbage/Transaction.hs

```diff
@@ -93,7 +93,7 @@ hprop_transaction = H.integrationRetryWorkspace 0 "babbage-transaction" $ \tempA
   -- This is the current calculated fee.
   -- It's a sanity check to see if anything has
   -- changed regarding fee calculation.
-  228 H.=== txFee
+  330 H.=== txFee
```

Comment:
Interesting, the fee increased. Can you put a comment saying that the previous fee was 228 lovelace?

Reply:
reverted